bindata/assets/openshift-controller-manager/old-leader-role.yaml

```diff
@@ -1,24 +0,0 @@
-# needed to get the legacy lock that we used to use
```

Comment:
We cannot remove these files yet. We first have to delete them in existing clusters with using:

https://github.com/openshift/cluster-openshift-controller-manager-operator/blob/0ab30996207ada63ef2e2fd4f47aac03b0872c9a/pkg/operator/starter.go#L246

The files can be removed only in the next major version of openshift at earliest.

Reply:
I moved the files into that section. Now sure how this works, I understood that you simply move the files into the other section, so that's what I did in https://github.com/openshift/cluster-openshift-controller-manager-operator/pull/385/commits/3cc19e789dde17919d3d2cf505eb88dd811a9ded . Also rolled back the change that removed the files, naturally.